docs/src/main/asciidoc/intro.adoc

```diff
@@ -164,6 +164,34 @@ include::https://raw.githubusercontent.com/spring-cloud-samples/sleuth-documenta
 NOTE: If you're using a custom `logback-spring.xml` then you have to pass the `spring.application.name` in
 `bootstrap` instead of `application` property file. Otherwise your custom logback file won't read the property properly.
 
+===== Propagating Span Context
+
+The span context is the state that must get propagated to any child Spans across process boundaries.
+Part of the Span Context is the Baggage.
+
+Baggage is a set of key:value pairs stored in the span context. Baggage travels together with the trace
+and is attached to every span. Spring Cloud Sleuth will understand that a header is baggage related if the HTTP
+ header is prefixed with `baggage-` and for messaging it starts with `baggage_`.
+
+IMPORTANT: Since you can attach as many baggage elements as you want and it's propagated through the network
```

Comment:
There's currently no limitation of the count or size of baggage items. However, keep in mind that too many can decrease system throughput or increase RPC latency. In extreme cases, it could crash the app due to exceeding transport-level message or header capacity.